ucm2/MediaTek/mt8390-evk/HiFi.conf

```diff
@@ -0,0 +1,168 @@
+SectionDevice."HDMI1" {
+	Comment "HDMI output"
+
+	Value {
+		PlaybackPriority 300
+		PlaybackPCM "hw:${CardId},5"
+		JackControl "HDMI Jack"
+	}
+
+	EnableSequence [
+		cset "name='HDMI_OUT_MUX' 1"
+	]
+
+	DisableSequence [
+		cset "name='HDMI_OUT_MUX' 0"
+	]
+}
+
+SectionDevice."HDMI2" {
```

Comment:
Missing conflict with HDMI1 (shared PCM device).

Reply:
I did not set HDMI1 and HDMI2 as conflicting devices because the platform supports scenarios where both HDMI1 and HDMI2 can be used simultaneously.

Reply:
OK. It makes sense. The priorities may be different.